frontend/server/src/DAO/GroupsIdentities.php

```diff
@@ -38,6 +38,8 @@ public static function getMemberIdentities(\OmegaUp\DAO\VO\Groups $group) {
                 States s ON s.state_id = i.state_id AND s.country_id = i.country_id
             LEFT JOIN
                 Countries c ON c.country_id = s.country_id
+            LEFT JOIN
+                Countries ci ON ci.country_id = i.country_id
```

Comment:
```suggestion
            LEFT JOIN
                Countries c ON c.country_id = i.country_id
```

un `JOIN` debería ser suficiente: no hay manera de que el nuevo `JOIN` contradiga lo que dice el otro si ambos están definidos, y los `IFNULL` son equivalentes a sólo consultar `ci`.